src/index.css

```
@@ -1,57 +1,76 @@
 }
 
-html,
-body {
-  background-color: white;
-  scroll-behavior: smooth;
-  scroll-padding-top: 100px;
-  box-sizing: border-box;
-  width: 100%;
-  /* nav 100px */
-  height: calc(100vh - 100px);
+@theme inline {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)